package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "strong-config-ts",
+  "version": "0.0.2",
+  "description": "SOPS-based secret management tool for NodeJS",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
+  "repository": "git@github.com:strong-config/node.git",
+  "author": "mohoff <moritz.hoffmann@brickblock.io>",
```

let's make this a team credit

```suggestion
  "author": "Brickblock Engineering <dev@brickblock.io>",
```